.packit.yaml

```
@@ -12,6 +12,8 @@ jobs:
   metadata:
     targets:
     - fedora-all
+    additional_repos:
+    - "http://kojipkgs.fedoraproject.org/repos/f$releasever-build/latest/$basearch/"
```

Ouch, this should also work and is probably better:

```suggestion
    - "https://kojipkgs.fedoraproject.org/repos/f$releasever-build/latest/$basearch/"
```